components/Menu/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Generated with scripts/create-component.js
+import { FC } from 'react';
+
+export interface MenuItemProps {
+  id: string | number;
+  content: React.ReactNode;
+  handleClick: Function;
```

Comment:
```suggestion
  onClick: Function;
```
onClick seems to make more sense for those who use the component

Reply:
I was thinking here and I believe that this prop doesn't even make much sense to stay in the items object. What about including it as a prop (onClick: (item) => {}) of the component instead of placing it inside the object?

Reply:
I think your suggestion adds a bit more complexity and my implementation with object literals is superior in this case making the code neater for the user to implement